BitwardenShared/UI/Tools/Generator/Generator/GeneratorState.swift

```diff
@@ -486,3 +497,44 @@ extension GeneratorState {
         return groups
     }
 }
+
+/// extension for `GuidedTourStepState` to provide states for learn new login guided tour.
+extension GuidedTourStepState {
+    /// The first step of the learn new login guided tour.
+    static let generatorStep1 = GuidedTourStepState(
+        arrowHorizontalPosition: .left,
+        spotlightShape: .rectangle(cornerRadius: 25),
+        title: Localizations.useTheGeneratorToCreateASecurePasswordPassphrasesAndUsernames
+    )
+
+    /// The first step of the learn new login guided tour.
```

Comment:
⛏️ Can you also add docs for the other steps?
```suggestion
    /// The first step of the generator guided tour.
    static let generatorStep1 = GuidedTourStepState(
        arrowHorizontalPosition: .left,
        spotlightShape: .rectangle(cornerRadius: 25),
        title: Localizations.useTheGeneratorToCreateASecurePasswordPassphrasesAndUsernames
    )

    /// The second step of the generator guided tour.
```